drf_spectacular/openapi.py

```diff
@@ -435,6 +435,8 @@ def _map_model_field(self, model_field, direction):
         elif hasattr(models, 'JSONField') and isinstance(model_field, models.JSONField):
             # fix for DRF==3.11 with django>=3.1 as it is not yet represented in the field_mapping
             return build_basic_type(OpenApiTypes.OBJECT)
+        elif hasattr(models, 'BinaryField') and isinstance(model_field, models.BinaryField):
+            return build_basic_type(OpenApiTypes.STR)
```

Comment:
This seems wrong to me. [`BinaryField`](https://docs.djangoproject.com/en/stable/ref/models/fields/#binaryfield) is documented as storing raw binary data. That implies that we should be using `OpenApiTypes.BINARY`:

https://github.com/tfranzel/drf-spectacular/blob/508f4e5f6ae94a53371e651417afe5fb771ee551/drf_spectacular/types.py#L68

*If* this were Base64 encoded we should be using `OpenApiTypes.BYTE`:

https://github.com/tfranzel/drf-spectacular/blob/508f4e5f6ae94a53371e651417afe5fb771ee551/drf_spectacular/types.py#L67

See the [data types](https://spec.openapis.org/oas/v3.0.3.html#data-types) section for OpenAPI v3.0.3.

Reply:
You are right! I fixed for OpenApiTypes.BYTE

Reply:
You've misunderstood me - this should be using `OpenApiTypes.BINARY` as `BinaryField` doesn't store Base64 encoded data, it stores raw binary data.

Reply:
I am confused. JSON format should be base64 type. I fixed for OpenApiTypes.BINARY.